api/v1alpha1/webhook_suite_test.go

```diff
@@ -103,6 +103,7 @@ var _ = BeforeSuite(func() {
 	//+kubebuilder:scaffold:webhook
 
 	go func() {
+		defer GinkgoRecover()
```

Comment:
Why is this needed?

Reply:
It allows ginkgo to catch failures from goroutines.  See https://onsi.github.io/ginkgo/#mental-model-how-ginkgo-handles-failure

I'm not sure that it matters much in this particular go routine.  I mainly just noticed that this is different from our other repos.